rules/linux/persistence_credential_access_modify_auth_module_or_config.toml

```diff
@@ -0,0 +1,92 @@
+[metadata]
+creation_date = "2020/12/21"
+maturity = "production"
+updated_date = "2021/03/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries may modify the standard authentication module for persistence via patching the normal authorization process
+or modifying the login configuration to allow unauthorized access or elevate privileges.
+"""
+false_positives = [
+    "Trusted system module updates or allowed Pluggable Authentication Module (PAM) daemon configuration changes.",
+]
+from = "now-9m"
+index = ["auditbeat-*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Modification of Standard Authentication Module or Configuration"
+note = """## Config
+
+This rule requires an additional setup in Auditbeat to log changes to the files specified in the query.
+
+In the file_integrity module on Auditbeat configuration file, add these values to the paths section:
+
+```
+  - /etc/pam.d
+```
+"""
```

Comment:
We can suggest changes to auditbeat configurations in file_integrity or audit_rules section. I'll be using file_integrity to paths and audit_rules for specific files